addons/mail/static/src/new/thread/attachment_list.xml

```diff
@@ -0,0 +1,87 @@
+                    }"
+                    t-att-title="attachment.name"
+                    tabindex="0"
+                    aria-label="View image"
+                    role="menuitem"
+                    t-att-data-mimetype="attachment.mimetype"
+                >
+                    <img t-if="!attachment.uploading" class="img img-fluid my-0 mx-auto" t-att-src="getImageUrl(attachment)" t-att-alt="attachment.name" t-attf-style="max-width: min(100%, {{ imagesWidth }}px); max-height: {{ props.imagesHeight }}px;" />
+                    <div t-if="attachment.uploading" class="position-absolute top-0 bottom-0 start-0 end-0 d-flex align-items-center justify-content-center" title="Uploading">
+                        <i class="fa fa-spin fa-spinner" />
+                    </div>
+                    <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column">
+                        <div t-if="canDelete(attachment)"
+                            class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
+                            t-att-class="{'o-pretty': env.inComposer }" tabindex="0" aria-label="Remove" role="menuitem" t-on-click="() => this.onClickUnlink(attachment)" title="Remove">
+                            <i class="fa fa-trash" />
+                        </div>
+                        <div t-if="canDownload(attachment)" class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover mt-auto" t-on-click="() => this.onClickDownload(attachment)" title="Download">
+                            <i class="fa fa-download" />
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="d-flex flex-grow-1 flex-wrap mt-1">
+                <div t-foreach="nonImagesAttachments" t-as="attachment" t-key="attachment.id"
+                    class="o-mail-attachment-card-aside o-has-card-details d-flex rounded bg-300 mb-1 me-1 mw-100"
+                    t-att-class="{
+                        'o-isUploading': attachment.uploading,
+                        'o-viewable': attachment.isViewable,
+                        'ms-1': isInChatWindowAndIsAlignedRight,
+                        'me-1': !isInChatWindowAndIsAlignedRight,
+                    }" t-att-title="attachment.name ? attachment.name : undefined" role="menu" t-att-aria-label="attachment.name"
+                >
+                    <!-- Image style-->
+                    <!-- o_image from mimetype.scss -->
+                    <div class="o-mail-attachment-image o_image flex-shrink-0 m-1" t-on-click="onClickImage" t-att-class="{'o-attachment-viewable opacity-75-hover': attachment.isViewable,}" role="menuitem" aria-label="Preview" t-att-tabindex="attachment.isViewable ? 0 : -1" t-att-aria-disabled="!attachment.isViewable" t-att-data-mimetype="attachment.mimetype" />
```

Comment:
`Attachment.isViewable` is not defined, so dead code. I'm removing it as feature is not yet implemented